signac/contrib/job.py

```diff
@@ -828,13 +828,13 @@ def fn(self, filename):
         Returns
         -------
         str
-            The full workspace path of the file.
+            The absolute path of the file.
 
         """
         return os.path.join(self.path, filename)
 
     def isfile(self, filename):
-        """Return True if file exists in the job's workspace.
+        """Check if a filename exists in the job path.
```

Comment:
Do files exist "in" paths?

Perhaps the formal way would be "Check if a file exists in the directory at the job path", but that's long.

And it might just be simpler to stick with one word (path)

```suggestion
        """Check if a filename exists in the job directory.
```

Reply:
@cbkerr Please feel free to apply changes to definitions / grammar in separate PRs, but this is far from the intended scope of this PR's focus on a singular API change. I tried to clean up a few egregious issues in the docs that seemed related to "root", but I am not able to invest the time needed to fully revamp the docs nor do I think it should be done in this PR.

Reply:
If you're okay with limiting the scope here, you may apply the suggestions you've already made on this PR but please keep it limited. I'm only able to commit a small amount of time to the project and multiple iterations on things like this make it hard to keep PRs small and focused.